dashboard/app/helpers/levels_helper.rb

```diff
@@ -379,7 +379,7 @@ def app_options
       @app_options[:experiments] =
         Experiment.get_all_enabled(user: current_user, section: section, script: @script).pluck(:name)
       @app_options[:usingTextModePref] = !!current_user.using_text_mode
-      @app_options[:muteMusic] = !!current_user.mute_music
+      @app_options[:muteMusic] = current_user.mute_music?
```

Comment:
You may also need to send this down in `ApiController#user_app_options` somewhere.  (Sorry, this code isn't very well factored at the moment.)